ajax/libs/jsnes/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jsnes",
+  "description": "A JavaScript NES emulator",
+  "author": "Ben Firshman <ben@firshman.co.uk>",
```

Comment:
Let's add website info to follow the upstream
`Ben Firshman <ben@firshman.co.uk> (https://fir.sh)`.